python/src/lib.rs

```diff
@@ -66,7 +63,11 @@ impl PyBallistaBuilder {
     /// Construct the standalone instance from the SessionContext
     pub fn standalone(&self, py: Python) -> PyResult<PySessionContext> {
         // Build the config
-        let config: SessionConfig = SessionConfig::from_string_hash_map(&self.conf)?;
+        let mut config: SessionConfig = SessionConfig::new_with_ballista();
```

Comment:
Did you update something in `new_with_ballista`? When I initially tried this I was getting an error. If it works thatn that is great!

Reply:
Changed nothing